editor/README.md

```diff
@@ -7,8 +7,391 @@ The logic flow concerning the editor includes: inferring a block representation
 
 The goal of the editor element is to let the user manipulate the content of their posts in a deterministic way—organized through the presence of blocks of content. Usually, in a declarative flow, the pieces that compose a post would be represented in a certain order and the machine would be able to generate an output view from it with the necessary UI controls. However, we don’t begin in WordPress with a representation of the state of the post that is conductive to this expression nor one that even has any knowledge of blocks because content is stored in a serialized way in a single field.
 
-Such a crucial step is handled by the grammar parsing which takes the serialized content of the post and infers an ordered block list using, preferably, syntax hints present in HTML comments. The editor is initialized with a state representation of the block nodes generated by the parsing of the raw content of a post element: `wp.blocks.parse( post.content.raw )`.
+Such a crucial step is handled by the grammar parsing which takes the serialized content of the post and infers an ordered block list using, preferably, syntax hints present in HTML comments. The editor is initialized with a state representation of the block nodes generated by the parsing of the raw content of a post element: `wp.editor.parse( post.content.raw )`.
```

Comment:
Some of these API conversions don't translate too well. Is `wp.editor.parse` as clear as `wp.blocks.parse` in what it's doing? Maybe... do we need to be more specific with block terminology in the exported names? `wp.editor.parseBlocks` ?

I'm on the fence.

Reply:
No strong arguments, and honestly, I'm not sure this will stay this way once we completely refactor the `editor` with the generic components approach.

Reply:
We had the similar issues before, e.g. the method like `wp.blocks.getCategories`. We can iterate later.